harmoni_core/harmoni_common_lib/src/harmoni_common_lib/helper_functions.py

```diff
@@ -140,10 +140,42 @@ def check_if_sensor(service_name):
             return True
     return False
 
+def topic_active(topic, msg_type):
+    """Determines if a topic has been published recently
```

Comment:
Does this actually check if it has been published recently? Or just published at all? A brief note on what the use is good for might be helpful.

Reply:
I guess it's recent in terms of ROS time. I don't know what the timeout is. Basically if a topic would show in `rostopic list` it would count as active here.